<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capture</class>
 <widget class="QWidget" name="Capture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resource/logo.ico</normaloff>resource/logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网页捕获音视频</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网址：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入想要捕获的网址</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下载地址：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_path">
     <property name="text">
      <string>选择下载地址</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>开始捕获</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
